config/routes.rb

```diff
@@ -3,6 +3,12 @@
   resource :cart, only: [:show]
   resources :staches, only: [:index, :show]
   resources :categories, only: [:index, :show]
+  resources :users, only: [:new, :create]
+
+  get '/login', to: "sessions#new"
+  post '/login', to: "sessions#create"
+  delete '/logout', to: "sessions#destroy"
+  get '/dashboard', to: "users#show"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.